templates/questions/show.html.twig

```diff
@@ -34,6 +34,24 @@
                                     </div>
                                 </form>
                             </div>
+                            {# Remove tags #}
+                            {% if is_granted('EDIT', question) %}
+                                <form action="{{ path('app_question_publish', { slug: question.slug }) }}" method="POST">
+                                    <div class="mt-2">
+                                        <h3>Tags:</h3>
+                                        {% for questionTag in question.questionTags %}
+                                            <a href="{{ path('app_question_remove_tag', {
+                                                id: questionTag.id,
+                                            }) }}" class="badge rounded-pill bg-light text-dark">
+                                            {{ questionTag.tag }}
+                                            </a>
+                                        {% endfor %}
+                                    </div>
+                                    {% if question.askedAt == null %}
```

Comment:
I think the form should live within this conditional check

Reply:
Done